README.md

```diff
@@ -50,12 +50,14 @@ Main dependencies in use:
 | Angular-CLI      | v10.0.0 | June 2020   |                                                   |
 | Electron         |  v9.0.0 | June 2020   | (internally uses Node `v12.14.1` and Chromium 83) |
 | Electron Builder | v22.7.0 | June 2020   |                                                   |
-
+|
```

Comment:
This might be accidental? It looks like this:

![image](https://user-images.githubusercontent.com/17264277/95103230-602fb380-0702-11eb-8b43-48dd302cd90a.png)